beep/structure.py

```diff
@@ -845,30 +1062,63 @@ def determine_structuring_parameters(self, v_range=None, resolution=1000,
                 finding and using the diagnostic cycles
 
         """
-        run_parameter, all_parameters = get_protocol_parameters(self.filename, parameters_path)
+        run_parameter, all_parameters = get_protocol_parameters(
+            self.filename, parameters_path
+        )
         # Logic for interpolation variables and diagnostic cycles
         diagnostic_available = False
         if run_parameter is not None:
-            if {'capacity_nominal'}.issubset(run_parameter.columns.tolist()):
-                nominal_capacity = run_parameter['capacity_nominal'].iloc[0]
-            if {'discharge_cutoff_voltage', 'charge_cutoff_voltage'}.issubset(run_parameter.columns):
-                v_range = [all_parameters['discharge_cutoff_voltage'].min(),
-                           all_parameters['charge_cutoff_voltage'].max()]
-            if {'diagnostic_type', 'diagnostic_start_cycle', 'diagnostic_interval'}.issubset(run_parameter.columns):
-                if run_parameter['diagnostic_type'].iloc[0] == 'HPPC+RPT':
-                    hppc_rpt = ['reset', 'hppc', 'rpt_0.2C', 'rpt_1C', 'rpt_2C']
+            if {"capacity_nominal"}.issubset(run_parameter.columns.tolist()):
+                nominal_capacity = run_parameter["capacity_nominal"].iloc[0]
+            if {"discharge_cutoff_voltage", "charge_cutoff_voltage"}.issubset(
+                run_parameter.columns
+            ):
+                v_range = [
+                    all_parameters["discharge_cutoff_voltage"].min(),
+                    all_parameters["charge_cutoff_voltage"].max(),
+                ]
+            if {
+                "diagnostic_type",
+                "diagnostic_start_cycle",
+                "diagnostic_interval",
+            }.issubset(run_parameter.columns):
+                if run_parameter["diagnostic_type"].iloc[0] == "HPPC+RPT":
+                    hppc_rpt = ["reset", "hppc", "rpt_0.2C", "rpt_1C", "rpt_2C"]
                     hppc_rpt_len = 5
-                    diagnostic_starts_at = [1, 1 + run_parameter['diagnostic_start_cycle'].iloc[0] + 1 * hppc_rpt_len]
+                    diagnostic_starts_at = [
+                        1,
+                        1
```

Comment:
This seems like it's a little hard to read.. Does flake8 complain if the addition is all in a single line?

Reply:
updated these two lines, flake8 wasn't complaining, but black reformatted this way.